checks/git_protocol.go

```diff
@@ -0,0 +1,120 @@
+package checks
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"time"
+
+	"github.com/flanksource/canary-checker/api/context"
+	"github.com/flanksource/canary-checker/api/external"
+	v1 "github.com/flanksource/canary-checker/api/v1"
+	"github.com/flanksource/canary-checker/pkg"
+	git "github.com/go-git/go-git/v5"
+	"github.com/go-git/go-git/v5/plumbing/object"
+	"github.com/go-git/go-git/v5/plumbing/transport/http"
+)
+
+const (
+	TestFileName = "test.txt"
```

Comment:
Can we move this to the check definition itself ?

Reply:
Done